defaults.yaml

```diff
@@ -20,6 +20,7 @@ net:
   minion_1_ipv6_part: "c41e::"
   minion_2_ipv6_part: "c41f::"
   minion_3_ipv6_part: "c420::"
+  if_name_for_flannel: enp0s8
```

Comment:
The interface name is not stable :(

We have to get it dynamically.

Reply:
ok, see my comment at the end